src/components/ResultList/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Grid, MenuItem, Typography } from '@mui/material';
+import { ChangeEvent, useState } from 'react';
+import SalonCard from '../SalonCard';
+import { CustomContainer, CustomSelect, CustomBox } from './resultList.styles';
+import { sortSelectData } from './sortSelectInput.data';
+import MOCK_DATA from './mock_data';
+
+const ResultList = () => {
+  const [sortType, setSortType] = useState<string | undefined>(undefined);
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setSortType(event.target.value);
+  };
+  return (
+    <CustomContainer disableGutters={true}>
+      <Grid container justifyContent="space-between" alignItems="center">
+        <Typography variant="h5">Results</Typography>
```

Comment:
I think you should additionally pass a component="h2" prop to transform it into an h2 tag.